plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -244,17 +259,13 @@ func (f *Fetcher) getRoles(ctx context.Context, opts []management.RequestOption)
 	return roles.Roles, roles.HasNext(), nil
 }
 
-func (f *Fetcher) getUserRoles(ctx context.Context, uID string) ([]map[string]interface{}, error) {
+func (f *Fetcher) fetchUserRoles(ctx context.Context, uID string) ([]map[string]any, error) {
 	page := 0
 	finished := false
 
-	var results []map[string]interface{}
-
-	for {
-		if finished {
-			break
-		}
+	var results []map[string]any
 
+	for !finished {
```

Comment:
It's always harder to read and understand negated expressions. I'd replace `finished` with `hasNext`. Then you won't have to negate anything and at the end of the loop you'll simply have:
```go
hasNext = roles.HasNext()
```